src/pages/algorithms/avl-trees/index.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: AVL Trees
+title: Avl Trees
```

Comment:
This changes `AVL` to `Avl` - do we need to update a proper nouns list or somehow mark that the all-caps version is the right way to display this?